src/screen/ApiLoadingScreen.tsx

```diff
@@ -1,21 +1,22 @@
-import React, {useContext} from 'react';
+import React, {useContext, useState} from 'react';
 import {ActivityIndicator, View, TouchableOpacity, StyleSheet} from 'react-native';
+import {useNavigation} from '@react-navigation/native';
 import {Button, Icon, Layout, Text} from '@ui-kitten/components';
 import globalStyles, {monofontFamily, colorGreen} from 'src/styles';
 import ScreenNavigation from 'layout/ScreenNavigation';
 import NetworkItem from 'presentational/NetworkItem';
 import {NetworkContext} from 'context/NetworkContext';
-import {NavigationProp} from '@react-navigation/native';
 import {ChainApiContext} from 'context/ChainApiContext';
+import NetworkSelect from 'src/layout/NetworkSelect';
 
-export function ApiLoadingScreen({navigation}: {navigation: NavigationProp<any>}) {
+export function ApiLoadingScreen() {
+  const navigation = useNavigation();
```

Comment:
why are we not using the navigation prop?

Reply:
I was going to move this screen to a normal component but decided not to go that path. It works the same